parser-typechecker/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -77,6 +77,9 @@ data Input
     | RemoveTermReplacementI PatchPath Reference Reference
     | RemoveTypeReplacementI PatchPath Reference Reference
   | UndoI
+  -- First `Maybe Int` is cap on number of results, if any
+  -- Second `Maybe Int` is cap on diff elements shown, if any
+  | LogI (Maybe Int) (Maybe Int) (Either Path' Branch.Hash)
```

Comment:
Minor inconsistency, `ForkLocalBranchI` takes a hash on the left and a path on the right; while `LogI` is the opposite.

Reply:
Fixed this, added an alias `Input.BranchId` for that either type, and a common parsing function `Input.parseBranchId` which parses it, and which is used in both input patterns that use this type (currently `fork` and `history`)